lib/common.sh

```diff
@@ -32,6 +33,23 @@ addAddresses() {
   printf %s "$result" > "$OUT_DIR/addresses.json"
 }
 
+dappBuild() {
+  test -n "$SKIP_BUILD" && return
+
+  local lib; lib=$1
+  (cd "$DAPP_LIB/$lib" || exit 1
+    dapp "${@:2}" build
+  )
+}
+
+dappCreate() {
+  local lib; lib=$1
+  local class; class=$2
+  DAPP_OUT="$DAPP_LIB/$lib/out" \
```

Comment:
Can we make this one line? I find it a bit confusing to follow otherwise.

Reply:
Sure thing.